openapi3.yaml

```
@@ -120,24 +120,25 @@ components:
         jobId:
           type: string
           description: job id
-    jobStatus:
+    delete-request:
```

Use the right convention:

```suggestion
    deleteRequest:
```

---

fixed